HelpMyStreet.Utils/HelpMyStreet.Utils/Extensions/RequestTypeExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using HelpMyStreet.Utils.Enums;
+using System;
+
+namespace HelpMyStreet.Utils.Extensions
+{
+    public static class RequestTypeExtensions
+    {
+        public static string PerfectTense(this RequestType requestType, int count)
```

Comment:
Perfect tense means past tense, so this method either needs to return e.g. "request completed", or it needs a different name.